src/scripts/main.js

```diff
@@ -0,0 +1 @@
+console.log('pesho');
```

Comment:
Have you tested this? Is the script working correctly? If you did, please delete line 1.